public/app/plugins/datasource/cloudwatch/components/SQLBuilderEditor/SQLBuilderEditor.tsx

```diff
@@ -65,6 +65,7 @@ export function SQLBuilderEditor({ query, datasource, onChange, onRunQuery }: Re
 
         <EditorField label="Limit" optional>
           <Input
+            aria-label="Limit"
```

Comment:
`aria-label` is not really needed here. If we add `id` to the `Input`, then the label will be automatically associated with it, making the `Input` accessible: https://github.com/grafana/grafana-experimental/blob/main/src/query-builder/EditorField.tsx#L25

Reply:
Thanks @Clarity-89, it's done. What do you think of the `id` I've added? I wasn't sure if there is a convention...
I also need to correct some other aria-labels I've added on `Input`s with existing labels.

Reply:
I don't think we really have any guidelines for input ids, the only issue I could see here if there's more than one editor rendered on a page, in which case the ids will duplicate. We have a helper hook to take care of that: https://github.com/grafana/grafana/blob/main/public/app/plugins/datasource/influxdb/components/useUniqueId.ts.

Reply:
Thanks! I found some other `id`s in the same plugin and followed their usage of the query ref ID in order to avoid the collision.